samples/bluetooth/unicast_audio_client/src/main.c

```diff
@@ -245,17 +256,25 @@ static void audio_timer_timeout(struct k_work *work)
 		data_initialized = true;
 	}
 
-	buf = net_buf_alloc(&tx_pool, K_FOREVER);
-	net_buf_reserve(buf, BT_ISO_CHAN_SEND_RESERVE);
+	/* We configured the sink streams to be first in `streams`, so that
+	 * we can use `stream[i]` to select sink streams (i.e. streams with
+	 * data going to the server)
+	 */
+	for (size_t i = 0; i < configured_sink_stream_count; i++) {
+		buf = net_buf_alloc(&tx_pool, K_FOREVER);
+		net_buf_reserve(buf, BT_ISO_CHAN_SEND_RESERVE);
```

Comment:
Looks good, but probably you could reuse the buffer, by just bumping its reference

Reply:
Oh yeah, that's actually a good idea since we are sending the same SDU on multiple CIS. Thanks!